docs/diagrams/sep6/withdraw.md

```diff
@@ -0,0 +1,47 @@
+### Withdraw
+
+This diagram illustrates the withdraw flow. The flow starts after the user has authenticated with the anchor via SEP-10 and provided some basic information about themselves. The example here shows the user withdrawing funds to their bank account, but the flow is similar for other withdrawal methods. The `withdraw-exchange` flow works similarly, but the Platform will additionally verify the quote requested or make a call to the Fee integration to update amounts.
+
+For more information on the withdraw flow, see the [SEP-6](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) specification.
+
+```mermaid
+sequenceDiagram
+  participant User
+  participant Wallet
+  participant Platform
+  participant Stellar
+  participant Anchor
+  participant Bank
+
+  Wallet->>+Platform: GET /withdraw
+  Platform->>Platform: Creates withdraw transaction with id abcd-1234 with status incomplete
+  Platform-->>-Wallet: Returns withdraw response with id abcd-1234
+  Platform-)Anchor: Sends an AnchorEvent with type transaction_created and transaction id abcd-1234
```

Comment:
Can we remove the references to `AnchorEvent`? This is more of an implementation detail, I think we can replace it with "event callback" or "webhook".